dbsc/workflow/rules/mrtpipelines.smk

```diff
@@ -1,571 +1,600 @@
 from os.path import join
 
-include: 'zona_bb_subcortex.smk'
-include: 'freesurfer.smk'
 
-# DWI Processing
-rule nii_to_mif:
+include: "zona_bb_subcortex.smk"
+include: "freesurfer.smk"
+
+
+# Directories
+mrtrix_dir = join(config["derivatives"], "mrtrix")
+
+# Paramaters
```

Comment:
Typo: `Paramaters` -> `Parameters`

Reply:
> I haven't actually tried running this yet, but a few comments from looking over the rules...

Haven't actually added this .smk to link with the target yet because there are still parts that aren't working in this workflow.